.gitignore

```diff
@@ -19,3 +19,5 @@ Cargo.lock
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+.env
```

Comment:
**nit**: Perhaps this is pedantic, but everything else from the `gitignore` template is commented, so this nags me a slight bit. I think removing all comments or adding a comment for this would be nice

Reply:
No worries, I chose to comment the `.env` entry!